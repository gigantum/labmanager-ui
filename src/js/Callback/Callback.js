import React, { Component } from 'react';
import Loader from 'Components/shared/Loader';

export default class Callback extends Component {
  render() {
    return(
      <div className="Callback">
        <div className="Callback__loader">
          <Loader />
        </div>
      </div>)
  }
}

