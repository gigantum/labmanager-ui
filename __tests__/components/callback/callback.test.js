import React from 'react';
import Callback from 'JS/callback/Callback';
import {shallow, mount} from 'enzyme'
import renderer from 'react-test-renderer';
import history from 'JS/history'

test('Test Overview rendering', () => {
  //const isAuthenticated = function(){return true};
  const component = renderer.create(
      <Callback history={history} />
  );

  let tree = component.toJSON();
  expect(component).toMatchSnapshot();

});
