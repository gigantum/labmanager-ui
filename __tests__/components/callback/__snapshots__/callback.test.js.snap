// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Overview rendering 1`] = `
<div
  className="Callback"
>
  <div
    className="Callback__loader"
  >
    <div
      className="Loader"
    />
  </div>
</div>
`;
